.travis.yml

```diff
@@ -25,6 +25,7 @@ matrix:
 before_install:
   - '[[ "$TRAVIS_PHP_VERSION" == "7.4snapshot" ]] || phpenv config-rm xdebug.ini'
   - composer self-update
+  - composer validate
```

Comment:
please move this to the `script` section, so that it is one of the tests and appear that way.